### PR TITLE
Add new documentation for using other networks.

### PR DESCRIPTION
**Pull request by NullSoldier**
https://website-git-jason-tesnet-ironfish.vercel.app/developers/documentation/networks

### CODE REVIEW

File: content/documentation/use-networks.mdx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+If you want to use the Devnet with the CLI run:
+
+```sh
+ironfish start --networkId 2
+```
+
+**Devnet Faucet**
+
+There is no, Devnet faucet. You can instantly mine a block using the built in CPU miner
```
Comment (hughy) [inline content/documentation/use-networks.mdx:40]:
```suggestion
There is no Devnet faucet. However, you can instantly mine a block using the built in CPU miner
```

File: content/documentation/use-networks.mdx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+```sh
+ironfish miners:start
+```
+
+# Mainnet
+
+All Iron Fish CLI, Node App, and SDK automatically connect to the main net by default. You don't need to do anything special to use it.
```
Comment (hughy) [inline content/documentation/use-networks.mdx:48]:
```suggestion
The Iron Fish CLI, Node App, and SDK each automatically connect to Mainnet by default. You don't need to do anything special to use it.
```

File: content/documentation/use-networks.mdx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+**Mainnet Faucet**
+
+There is no faucet for Mainnet, please use [Testnet](#testnet-0) for pre-production testing.
+
+# Custom Network
+
+Iron Fish supports custom network definitions. You can create a custom network definition and import it using the [customNetwork config](/use/get-started/node-configuration) or the -customNetwork flag like this...
```
Comment (hughy) [inline content/documentation/use-networks.mdx:56]:
```suggestion
Iron Fish supports custom network definitions. You can create a custom network definition and import it using the [customNetwork config](/use/get-started/node-configuration) or the `--customNetwork` flag like this...
```

File: content/documentation/use-networks.mdx [hunk 4]
```diff
@@ -0,0 +1,82 @@
+---
+title: Use Networks
```
Comment (hughy) [inline content/documentation/use-networks.mdx:2]: "Use Networks" seems a little odd to me. Maybe just "Networks"? "Iron Fish Networks"?

File: content/documentation/use-networks.mdx [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+description: Use Testnet | Iron Fish Documentation
+---
+
+Iron Fish has 3 networks built into the SDK by default. These networks are useful for development and testing rather than trying to wait for someone to send you Mainnet $IRON.
+
+> **Note:** Once you start a node on a certain network, that node cannot be used on another network. Delete the data directory in `ironfish browse` to reset the data directory.
```
Comment (hughy) [inline content/documentation/use-networks.mdx:8]: Should we also refer to `ironfish reset`?

File: content/documentation/use-networks.mdx (hunk 4 above)
Comment (NullSoldier) [reply content/documentation/use-networks.mdx]: Changed

File: content/documentation/use-networks.mdx (hunk 5 above)
Comment (NullSoldier) [reply content/documentation/use-networks.mdx]: I just tested that and it doesn't actually work to clear the right flags to let me switch networks. We should fix that, I'll file a linear issue.

File: content/documentation/integration_local.mdx [hunk 6]
```diff
@@ -8,6 +8,8 @@ network rather than using the testnet or mainnet and waiting for funds. Iron
 Fish has a devnet with low difficulty which enables easy block mining and
 transfers.
 
+You can read more about our included networks in the [networks documentation](/developers/documentation/use-networks).
```
Comment (hughy) [inline content/documentation/integration_local.mdx:11]:
```suggestion
You can read more about our included networks in the [networks documentation](/developers/documentation/networks).
```

**Review by hughy — APPROVED** [review]
looks good pending link change!

File: content/documentation/use-networks.mdx (hunk 5 above)
Comment (NullSoldier) [reply content/documentation/use-networks.mdx]: Ok fixed

**Pull request merged**